components/authz-service/storage/postgres/migration/legacy/v1datamodel.go

```diff
@@ -0,0 +1,269 @@
+package legacy
```

Comment:
The data model files represent the v1 and v2 data structures at the time of force upgrade